aiida/manage/manager.py

```diff
@@ -9,11 +9,12 @@
 ###########################################################################
 # pylint: disable=cyclic-import
 """AiiDA manager for global settings"""
-import asyncio
```

Comment:
Note: About half of the cost of importing plumpy (30ms) comes from importing `asyncio`